tachyon/crypto/commitments/kzg/kzg.h

```diff
@@ -61,13 +61,9 @@ class KZG {
     std::vector<Field> powers_of_tau = Field::GetSuccessivePowers(size, tau);
     std::vector<G1JacobianPoint> g1_powers_of_tau_jacobian;
 
-    g1_powers_of_tau_jacobian.resize(size);
-    if (!G1Point::MultiScalarMul(powers_of_tau, g1,
-                                 &g1_powers_of_tau_jacobian)) {
-      return false;
-    }
     g1_powers_of_tau_.resize(size);
-    if (!math::ConvertPoints(g1_powers_of_tau_jacobian, &g1_powers_of_tau_)) {
+    if (!G1Point::BatchMapScalarFieldToPoint(g1, powers_of_tau,
+                                             &g1_powers_of_tau_)) {
       return false;
     }
```

Comment:
```suggestion
    if (!G1Point::BatchMapScalarFieldToPoint(g1, powers_of_tau, &g1_powers_of_tau_)) return false;
```
I'm not sure we have unified code in our repo like above suggestion, but I think I've seen this style more frequently. How about fixing like this?

Reply:
IMHO, it's up to developers for the things not described in google style if this doesn't impact on performance!